test/runtests.sh

```diff
@@ -60,6 +68,10 @@ else
 	TESTS="$(join , "${ARGS[@]}")"
 fi
 
+if [ "$TRANSLATOR_TESTS" = true ]; then
+	TESTS="translatorDependentFeatures"
```

Comment:
I don't want to move all translator-related tests into a single file. Lookup tests should stay in lookup.js and be runnable via 'lookup'. I think we can either do something with a subdirectory or just keep a hard-coded list of translator-related files in runtests.js. (We'll just need some mechanism for excluding those.)